build.gradle

```diff
@@ -202,5 +241,43 @@ android {
               systemProperty 'robolectric.logging', 'stdout'
         }
     }
+}
 
+/**
+ * Download and unpack commcare app associated with 'cc_app_id' into assets
+ * folder
+ */
+task downloadCCApp(type: Exec)
```

Comment:
Think this is a (necessary) declaration. `downloadCCApp` is still being used to trigger the acquisition of the ccz from HQ via a `wget` call